downstream/snippets/cont-tested-system-config.adoc

```diff
@@ -1,14 +1,31 @@
 //Tested system configuration snippet for container (CONT) topologies
 [options="header"]
 |====
-| Type | Description 
+| Type | Description | Notes
 | Subscription 
 a| 
 * Valid {PlatformName} subscription
 * Valid {RHEL} subscription (to consume the BaseOS and AppStream repositories)
-| Operating system | {RHEL} 9.2 or later minor versions of {RHEL} 9
-| CPU architecture | x86_64, AArch64, s390x (IBM Z), ppc64le (IBM Power)
-| Ansible-core | Ansible-core version {CoreInstVers} or later
-| Browser | A currently supported version of Mozilla Firefox or Google Chrome.
-| Database | {PostgresVers}
-|====
\ No newline at end of file
+|
+
+| Operating system 
+| {RHEL} 9.2 or later minor versions of {RHEL} 9
+| 
+
+| CPU architecture 
+| x86_64, AArch64, s390x (IBM Z), ppc64le (IBM Power)
+|
+
+| `ansible-core` 
+| `ansible-core` version {CoreUseVers} or later. 
+| {PlatformNameShort} uses the system-wide `ansible-core` package to install the platform, but uses `ansible-core` {CoreUseVers} for both its control plane and built-in execution environments. The required version is bundled within the platform's components, so you do not need to manually install it on your host system.
```

Comment:
Not sure to fully understand which ansible-core version and usage we're talking about so I'll try to explain to see if we're on the same page.

The containerized installer should use ansible-core 2.14 RPM package from RHEL 9 appstream repository (because we don't need to have AAP subscription enabled on the installer node).

The one from the AAP 2.5 repository shouldn't be used by the installer. It's possible but not mandatory. So running the installer with either 2.14 or 2.16 is working. Other later versions aren't verified.

However, the platform deployed by the installer will only use ansible-core 2.16 from AAP.